usaspending_api/etl/management/commands/combine_universal_transaction_matview_chunks.py

```diff
@@ -0,0 +1,200 @@
+import asyncio
+import sqlparse
+
+from django.db import connection
+from django.core.management.base import BaseCommand
+from usaspending_api.common.data_connectors.async_sql_query import async_run_creates
+from usaspending_api.common.helpers.timing_helpers import ConsoleTimer as Timer
+
+logger = logging.getLogger("script")
+
+RECREATE_TABLE_SQL = """
+DROP {old_object_type} IF EXISTS universal_transaction_matview_temp;
+CREATE TABLE universal_transaction_matview_temp AS SELECT * from universal_transaction_matview_0 WITH NO DATA;
+"""
+
+INSERT_INTO_TABLE_SQL = """
+INSERT INTO universal_transaction_matview_temp SELECT * FROM universal_transaction_matview_{current_chunk}
+"""
+
+TABLE_INDEX_SQL = """
+CREATE UNIQUE INDEX idx_ut_transaction_id_temp ON universal_transaction_matview_temp USING BTREE(transaction_id ASC NULLS LAST) WITH (fillfactor = 97);
+CREATE INDEX idx_ut_action_date_temp ON universal_transaction_matview_temp USING BTREE(action_date DESC NULLS LAST) WITH (fillfactor = 97) WHERE action_date >= '2007-10-01';
+CREATE INDEX idx_ut_last_modified_date_temp ON universal_transaction_matview_temp USING BTREE(last_modified_date DESC NULLS LAST) WITH (fillfactor = 97);
+CREATE INDEX idx_ut_fiscal_year_temp ON universal_transaction_matview_temp USING BTREE(fiscal_year DESC NULLS LAST) WITH (fillfactor = 97) WHERE action_date >= '2007-10-01';
+CREATE INDEX idx_ut_type_temp ON universal_transaction_matview_temp USING BTREE(type) WITH (fillfactor = 97) WHERE type IS NOT NULL AND action_date >= '2007-10-01';
+CREATE INDEX idx_ut_award_id_temp ON universal_transaction_matview_temp USING BTREE(award_id) WITH (fillfactor = 97) WHERE action_date >= '2007-10-01';
+CREATE INDEX idx_ut_pop_zip5_temp ON universal_transaction_matview_temp USING BTREE(pop_zip5) WITH (fillfactor = 97) WHERE pop_zip5 IS NOT NULL AND action_date >= '2007-10-01';
+CREATE INDEX idx_ut_recipient_unique_id_temp ON universal_transaction_matview_temp USING BTREE(recipient_unique_id) WITH (fillfactor = 97) WHERE recipient_unique_id IS NOT NULL AND action_date >= '2007-10-01';
+CREATE INDEX idx_ut_parent_recipient_unique_id_temp ON universal_transaction_matview_temp USING BTREE(parent_recipient_unique_id) WITH (fillfactor = 97) WHERE parent_recipient_unique_id IS NOT NULL AND action_date >= '2007-10-01';
+CREATE INDEX idx_ut_simple_pop_geolocation_temp ON universal_transaction_matview_temp USING BTREE(pop_state_code, action_date) WITH (fillfactor = 97) WHERE pop_country_code = 'USA' AND pop_state_code IS NOT NULL AND action_date >= '2007-10-01';
+CREATE INDEX idx_ut_recipient_hash_temp ON universal_transaction_matview_temp USING BTREE(recipient_hash) WITH (fillfactor = 97) WHERE action_date >= '2007-10-01';
+CREATE INDEX idx_ut_action_date_pre2008_temp ON universal_transaction_matview_temp USING BTREE(action_date) WITH (fillfactor = 97) WHERE action_date < '2007-10-01';
+"""
+
+SWAP_TABLES_SQL = """
+DROP {old_object_type} IF EXISTS universal_transaction_matview_old;
```

Comment:
I understand why it was done this way. This is similar to the first SQL scripts for the materialized views. Updating and managing those files was a massive headache.

Is there any other solution to make the renaming more programatic so they are defined once and devs don't need to make three changes?